package.json

```diff
@@ -40,8 +35,18 @@
     "eslint-config-next": "^11.0.1",
     "husky": "^6.0.0",
     "jest": "^27.0.5",
+    "lint-staged": ">=10",
+    "organize-imports-cli": "^0.8.0",
```

Comment:
We'll have a hard time if this and the "organizing import" in VSCode out of sync https://github.com/thorn0/organize-imports-cli :))

Reply:
😓 yup, but I think it won't be a problem because we always use the cli script to organize the import statements